cloud/azure/variables.tf

```
@@ -20,6 +20,18 @@ variable "poll_rate" {
   default     = 300
 }
 
+variable "host_or_usage_limits" {
```

What about this ?

```suggestion
variable "usage_limits" {
```

having hosts is one kind of usage in my opinion.

---

this is a exclusive "or" it seems pretty obvious, I prefer to keep the original name which correspond to the official resource https://registry.terraform.io/providers/splunk-terraform/signalfx/latest/docs/resources/org_token from this way this is easy to understand for the user